src/main/webapp/WEB-INF/tags/instructor/results/noResponsePanel.tag

```diff
@@ -27,7 +27,12 @@
                 <tbody>
                     <c:forEach items="${noResponsePanel.emails}" var="email">
                         <tr>
-                            <td>${fn:escapeXml(noResponsePanel.teams[email])}</td>
+                            <c:if test="${not empty noResponsePanel.instructorStatus[email]}">
+                                     <td><i>${fn:escapeXml(noResponsePanel.teams[email])}</i></td>
+                            </c:if>
+                            <c:if test="${empty noResponsePanel.instructorStatus[email]}">
```

Comment:
@kanghj We add to that map only if it is an instructor email. We could have used a list also for this purpose. But checking whether a list contains a value in JSTL is bit tricky. This is why I went with a map. However, I made the positive check stronger by making it to check `not empty && equal true`.